src/android/java/io/jxcore/node/BtToRequestSocket.java

```diff
@@ -170,77 +170,92 @@ public int GetLocalHostPort() {
     }
 
     public String GetLocalHostAddress() {
-        String ret = "";
```

Comment:
Question - why is there a setPort method? We only seem to call it in one place and we always pass 0. Wouldn't it be easier to just include the port in the constructor? Or better yet, never include a port at all and always have it be 0? If we set it in the constructor then we can make mHTTPPort final and that is always a good thing.